packages/transactions/src/OutboundShipment/DetailView/DetailView.tsx

```diff
@@ -150,23 +155,53 @@ export const OutboundShipmentDetailView: FC = () => {
     return () => setActions([]);
   }, [draft]);
 
+  const { currentTab, onChangeTab } = useTabs(0);
+
   return draft ? (
     <>
       <Portal container={appBarButtonsRef?.current}>
         <>{OpenButton}</>
       </Portal>
-      <div>
-        <input
-          value={draft?.name}
-          onChange={event => setDraft({ ...draft, name: event?.target.value })}
-        />
-      </div>
-      <div>
-        <span>{JSON.stringify(draft, null, 4) ?? ''}</span>
-      </div>
-      <div>
-        <button onClick={save}>OK</button>
-      </div>
+      <AppBarContentPortal
+        sx={{ display: 'flex', flex: 1, justifyContent: 'center' }}
+      >
+        <Tabs value={currentTab} centered onChange={onChangeTab}>
+          <Tab label={t('label.general')} />
+          <Tab label={t('label.item')} />
+          <Tab label={t('label.batch')} />
+          <Tab label={t('label.price')} />
+          <Tab label={t('label.log')} />
+        </Tabs>
+      </AppBarContentPortal>
+
+      <TabPanel tab={0} currentTab={currentTab}>
+        <div>
```

Comment:
will need to lift the tab contents to their own components soon, this file is going to get huge

Reply:
For sure!